xonsh/xoreutils/_which.py

```diff
@@ -232,7 +231,7 @@ def whichgen(command, path=None, verbose=0, exts=None):
                     elif i == 0:
                         fromWhere = "from current directory"
                     else:
-                        fromWhere = "from PATH element %d" % (i-1)
+                        fromWhere = "from PATH element %d" % (i - 1)
```

Comment:
I think all the whitespace changes are OK. But the rest we really shouldn't change.